mac/06/_bugsnag_x86_64.crash

```diff
@@ -1,28 +1,25 @@
-Segmentation fault: 11: 
-
-bugsnag/BugsnagEvent.m:91:0 -[BugsnagEvent getStackTraceWithException:]
-bugsnag/BugsnagEvent.m:50:0 -[BugsnagEvent addSignal:]
-bugsnag/BugsnagNotifier.m:73:0 -[BugsnagNotifier notifySignal:]
-bugsnag/Bugsnag.m:67:0 handle_signal
-libsystem_platform.dylib:0 sigtramp
-<span class="cp-wrong">CrashLib:0 _31-[CRLCrashReleasedObject crash]_block_invoke | Missing filename, and line number</span>
-<span class="cp-wrong">Missing frame</span>
-CrashProbe/CRLMainWindowController.m:72:0 -[CRLMainWindowController causeCrash:]
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSApplication sendAction:to:from:]
-AppKit:0 -[NSControl sendAction:to:]
-AppKit:0 _26-[NSCell _sendActionFrom:]_block_invoke
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSCell _sendActionFrom:]
-AppKit:0 -[NSButtonCell _sendActionFrom:]
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSCell trackMouse:inRect:ofView:untilMouseUp:]
-AppKit:0 -[NSButtonCell trackMouse:inRect:ofView:untilMouseUp:]
-AppKit:0 -[NSControl mouseDown:]
-AppKit:0 -[NSWindow _reallySendEvent:]
-AppKit:0 -[NSWindow sendEvent:]
-AppKit:0 -[NSApplication sendEvent:]
-AppKit:0 -[NSApplication run]
-AppKit:0 NSApplicationMain
-CrashProbe/main.m:13:0 main
-libdyld.dylib:0 start
+Exception Type:  EXC_BAD_ACCESS (SIGBUS)
+Exception Subtype: KERN_MEMORY_DATA_MOVED
+                                                                                                   
+0  libobjc.A.dylib       objc_msgSend                                                          
+1  CrashProbe            -[CRLCrashReleasedObject crash] (CRLCrashReleasedObject.m:49)         
```

Comment:
Gotcha. I went by the [Apple report](http://www.crashprobe.com/mac/06/), which was marked correct but also missing the block frame. It might be out of sync with the latest [report data](https://github.com/bitstadium/CrashProbe-Data/blob/3d9f953294a4344ed19fc5f2b8a476671b66a500/ios/06/_apple_arm64.crash), where it appears correctly. Patching.